lib/icedfrisby.js

```diff
@@ -676,8 +681,13 @@ class Frisby {
     header = (header + "").toLowerCase()
     this.current.expects.push(function () {
       if (typeof self.current.response.headers[header] !== "undefined") {
-        chai.expect(self.current.response.headers[header].toLowerCase()).to
-          .match(pattern)
+        if(self.current.response.headers[header] instanceof Array){
+          chai.expect(self.current.response.headers[header]).to
+            .include.something.that.matches(pattern)
+        } else {
+          chai.expect(self.current.response.headers[header].toLowerCase()).to
+            .match(pattern)
```

Comment:
Would it be more concise / simpler to coerce `self.current.response.headers[header]` to an array, and then avoid the `if` statement? I think the failure message would be equally clear.

Reply:
Fixed.